src/control/server/storage_nvme.go

```diff
@@ -112,6 +115,39 @@ func (s *spdkSetup) prep(nrHugepages int, usr string) error {
 		hPages, tUsr, stderr.String())
 }
 
+// prep executes setup script to allocate hugepages and unbind specified PCI
+// devices (that don't have active mountpoints) from generic kernel driver to be
+// used with SPDK.
+//
+// NOTE: will make the controller disappear from /dev until reset() called.
+func (s *spdkSetup) prep(nrHugepages int, usr string, wlist string) error {
```

Comment:
probably want to consolidate the two commands, prepAll could call prep with empty wlist string and prep only populate wlist env if wlist parameter is not empty? does that make sense?

Reply:
yes definitely I can simplify the code, just wanted to make sure that this seemed like a good addition to prep-nvme command?

Reply:
yes looks good, might be worth adding a description of what the whitelist syntax looks like when it's not just a single PCI 
 address